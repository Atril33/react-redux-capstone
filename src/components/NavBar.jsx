import { Link } from 'react-router-dom';
import back from './images/back.png';
import microphone from './images/microphone.png';
import setting from './images/setting.png';

const NavBar = () => {
  return (
    <div className='top-header'>
     <Link to='/'><img src={back} alt='Back Icon' className='back-img'></img></Link> 
      <h1>Countries</h1>
      <div className='right-icon'>
         <img src={microphone} alt='Microphone icon'></img>
         <img src={setting} alt='Setting icon' className='setting-img'></img>
      </div>
    </div>
  )
}

export default NavBar;