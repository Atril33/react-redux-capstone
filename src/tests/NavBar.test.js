import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../components/NavBar';

describe('NavBar', () => {
  it('Should renders correctly', () => {
    const component = renderer.create(
      <Router>
        <NavBar />
      </Router>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
