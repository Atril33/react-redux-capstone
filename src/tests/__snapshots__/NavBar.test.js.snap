// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Should renders correctly 1`] = `
<div
  className="top-header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Back Icon"
      className="back-img"
      src="back.png"
    />
  </a>
  <h1>
    Countries
  </h1>
  <div
    className="right-icon"
  >
    <img
      alt="Microphone icon"
      src="microphone.png"
    />
    <img
      alt="Setting icon"
      className="setting-img"
      src="setting.png"
    />
  </div>
</div>
`;

exports[`NavBar renders correctly 1`] = `
<div
  className="top-header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Back Icon"
      className="back-img"
      src="back.png"
    />
  </a>
  <h1>
    Countries
  </h1>
  <div
    className="right-icon"
  >
    <img
      alt="Microphone icon"
      src="microphone.png"
    />
    <img
      alt="Setting icon"
      className="setting-img"
      src="setting.png"
    />
  </div>
</div>
`;
